packages/custom_lint/lib/src/v2/custom_lint_analyzer_plugin.dart

```diff
@@ -49,10 +49,18 @@ class CustomLintServer {
       },
       (err, stack) => server.handleUncaughtError(err, stack),
       zoneSpecification: ZoneSpecification(
-        print: (self, parent, zone, line) => server.handlePrint(
-          line,
-          isClientMessage: false,
-        ),
+        print: (self, parent, zone, line) {
+          server.handlePrint(
+            line,
+            isClientMessage: false,
+          );
+          // Send the print to the parent zone.
+          // Print is heavily used by the cli_util package.
+          // Additionally developers of plugins may wonder
+          // why their print statements are not showing up
+          // when not knowing about the log file.
+          parent.print(zone, line);
```

Comment:
This shouldn't be here. This should be inside server.handlePrint.

Reply:
Would have to pass the zone there somehow and then it is still run async which may mess with the output order.

Reply:
This is also a general question, should prints from plugins show up in the terminal or not.
The first time I used custom_lint, I was wondering why my prints were not showing up.

Reply:
This is removed now